packages/changed-elements-react/src/widgets/comparisonJobWidget/common/versionCompareV2WidgetUtils.ts

```diff
@@ -6,24 +6,37 @@ import { IModelConnection } from "@itwin/core-frontend";
 import { ComparisonJobCompleted, ComparisonJobStarted, IComparisonJobClient } from "../../../clients/IComparisonJobClient";
 import { NamedVersion } from "../../../clients/iModelsClient";
 import { VersionCompare } from "../../../api/VersionCompare";
-import { toastComparisonVisualizationStarting } from "./versionComapreToasts";
+import { toastComparisonVisualizationStarting } from "./versionCompareToasts";
 import { Logger } from "@itwin/core-bentley";
-import { JobStatusAndJobProgress } from "../models/ComparisonJobModels";
+import { JobAndNamedVersions, JobStatusAndJobProgress } from "../models/ComparisonJobModels";
 import { VersionState } from "../models/VersionState";
+import { ComparisonJobUpdateType } from "../components/VersionCompareDialogProvider";
 
 export type ManagerStartComparisonV2Args = {
   comparisonJob: ComparisonJobCompleted;
   comparisonJobClient: IComparisonJobClient;
   iModelConnection: IModelConnection;
   targetVersion: NamedVersion;
   currentVersion: NamedVersion;
+  getToastsEnabled: () => boolean;
+  runOnJobUpdate: (comparisonEventType: ComparisonJobUpdateType, jobAndNamedVersions?: JobAndNamedVersions) => Promise<void>;
 };
 
 export const runManagerStartComparisonV2 = async (args: ManagerStartComparisonV2Args) => {
   if (VersionCompare.manager?.isComparing) {
     return;
   }
-  toastComparisonVisualizationStarting();
+  if (args.getToastsEnabled()) {
+    toastComparisonVisualizationStarting();
```

Comment:
When you have some time, could you show me what this toast looks like? A screenshot would suffice. This toast may be irrelevant.

Reply:
<img width="420" alt="image" src="https://github.com/iTwin/changed-elements-react/assets/86487204/ada2e72e-d62d-44d8-863a-56c7e6021e5d">

Reply:
This is something i added because the comparisons can take a while to start after hitting start comparison

Reply:
Yes, but the widget does provide information on the loading state, right? (e.g. loading models, assemblies, etc)

Reply:
It does, but there is still delay, especially if there is a larger amount of changeset. Sometimes you click and you have to wait about 1-3 seconds to see any state change in the widget pannel.

Reply:
Hmm is it due to the download of the HREF?

Reply:
Yes:
When you click start comparison, it closes the dialog and grabs the elements from the href, but there is a delay sometimes, which is why I added the toast. Otherwise the user clicks and looks like nothing is happening when the href is downloading.

Reply:
Can this toast not be sticky? e.g. it should go away automatically instead of having to click on the `X`